bugbug/repository.py

```diff
@@ -965,11 +966,7 @@ def clone(repo_dir):
 
     cmd.insert(0, hglib.HGPATH)
 
-    proc = hglib.util.popen(cmd)
-    out, err = proc.communicate()
-    if proc.returncode:
-        raise hglib.error.CommandError(cmd, proc.returncode, out, err)
-
+    subprocess.check_output(cmd)
```

Comment:
With this change we'd lose the stdout, previously it was included in the exception

Reply:
Sorry, the stderr